includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1114,6 +1114,9 @@ public function process_payment_for_order( $cart, $payment_information, $schedul
 				$request->set_off_session( $payment_information->is_merchant_initiated() );
 				$request->set_payment_methods( $payment_methods );
 				$request->set_cvc_confirmation( $payment_information->get_cvc_confirmation() );
+				if ( Payment_Method::AFTERPAY === $payment_information->get_payment_method() ) {
```

Comment:
We need a short comment that Afterpay differs from other PMs and expects shipping details to be supplied.

Reply:
Added [here](https://github.com/Automattic/woocommerce-payments/commit/242cb56552dbfcbe5897ca39e0422518aa6f98d6#diff-bcfd303d507c33a27671935e724ee64d0df30ff2a18bb6ef4c94f28d48190124R1118), thanks.

Reply:
Added[ here ](https://github.com/Automattic/woocommerce-payments/commit/242cb56552dbfcbe5897ca39e0422518aa6f98d6#diff-bcfd303d507c33a27671935e724ee64d0df30ff2a18bb6ef4c94f28d48190124R1118)